src/NuGetGallery.Core/GitHub/NuGetPackageGitHubInformation.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Collections.Generic;
+
+namespace NuGetGallery.GitHub
+{
+    public class NuGetPackageGitHubInformation
+    {
+        public NuGetPackageGitHubInformation() : this(0, null)
```

Comment:
It looks like this constructor will throw an exception due to `null` check in the other constructor